### PR TITLE
Add 'Where did you hear about Pension Wise' to appointments/new

### PR DESCRIPTION
**Pull request by AndrewVos**

### CODE REVIEW

File: app/helpers/appointment_helper.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+module AppointmentHelper
+  WHERE_DID_YOU_HEAR_ABOUT_PENSION_WISE_OPTIONS = [
```
Comment (benlovell) [inline app/helpers/appointment_helper.rb]: Are these ordered specifically?

File: db/migrate/20161011115911_add_extra_fields_to_appointments.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class AddExtraFieldsToAppointments < ActiveRecord::Migration[5.0]
+  def up
+    add_column :appointments, :where_did_you_hear_about_pension_wise, :string
+    execute 'UPDATE appointments SET where_did_you_hear_about_pension_wise = \'unknown\''
```
Comment (benlovell) [inline db/migrate/20161011115911_add_extra_fields_to_appointments.rb]: Maybe `unknown` should be the default and we could dispense with this?

File: app/helpers/appointment_helper.rb (hunk 1 above)
Comment (AndrewVos) [inline app/helpers/appointment_helper.rb]: I didn't ask, but they come from here:

https://trello.com/c/XZoLbCJZ/119-appointment-booking-part-2

I'll check with @mattlucht

Comment (benlovell) [inline app/helpers/appointment_helper.rb]: Cool.

File: db/migrate/20161011141405_add_where_did_you_hear_about_pension_wise_to_appointments.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class AddWhereDidYouHearAboutPensionWiseToAppointments < ActiveRecord::Migration[5.0]
+  def up
+    add_column :appointments, :where_did_you_hear_about_pension_wise, :string
+    execute 'UPDATE appointments SET where_did_you_hear_about_pension_wise = \'unknown\''
+    change_column_null :appointments, :where_did_you_hear_about_pension_wise, false
```
Comment (benlovell) [inline db/migrate/20161011141405_add_where_did_you_hear_about_pension_wise_to_appointments.rb]: You can just add this to the `add_column` declaration as `null: false` instead.

File: db/migrate/20161011115911_add_extra_fields_to_appointments.rb (hunk 2 above)
Comment (AndrewVos) [inline db/migrate/20161011115911_add_extra_fields_to_appointments.rb]: Cool cool done. Also added validation for that field.

File: db/migrate/20161011141405_add_where_did_you_hear_about_pension_wise_to_appointments.rb (hunk 3 above)
Comment (AndrewVos) [inline db/migrate/20161011141405_add_where_did_you_hear_about_pension_wise_to_appointments.rb]: done

File: app/helpers/appointment_helper.rb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    'Newspaper/Magazine advert',
+    'Friend/Word of mouth',
+    'Local Advertising',
+    'Job Centre',
+    'Charity'
+  ].freeze
+
+  def where_did_you_hear_about_pension_wise_options(appointment)
+    options = WHERE_DID_YOU_HEAR_ABOUT_PENSION_WISE_OPTIONS
+    options << appointment.where_did_you_hear_about_pension_wise if appointment.where_did_you_hear_about_pension_wise
```
Comment (AndrewVos) [inline app/helpers/appointment_helper.rb]: Good catch! Done

**Comment by benlovell** [comment]
:ship:

**Pull request merged**